bundles/org.openhab.binding.liquidcheck/README.md

```diff
@@ -0,0 +1,64 @@
+# LiquidCheck Binding
+
+This binding is for the Liquid-Check device from SI-Elektronik GmbH which can be used to measure level and content of tanks.
+
+## Supported Things
+
+`liquidCheckDevice`:
+
+The Liquid-Check device in Hardwareversion B and Firmwareversion 1.60 has been tested and is working.
+You can access the measured data, raw data, the settings as properties and command a measurement.
+
+## Discovery
+
+This binding discovers the devices via a ping and request method.
+It uses every Ethernet/WLAN interface that is connected to the openHAB server.
+Therefore the discovery has to be manually triggered.
+
+## Thing Configuration
+
+You only need to set the IP address of the device or use the discovery method.
+If the maximum content has not been set the fill-indicator channel will not contain valid values.
+
+### `liquidCheckDevice` Thing Configuration
+
+| Name             | Type    | Description                           | Default | Required | Advanced |
+|------------------|---------|---------------------------------------|---------|----------|----------|
+| hostname         | text    | Hostname or IP address of the device  | N/A     | yes      | no       |
+| maxContent       | integer | Maximum Content of the tank           | 1       | no       | no       |
```

Comment:
Consistency with thing parameter description:
```suggestion
| maxContent       | integer | Maximal content in the container      | 1       | no       | no       |
```

Reply:
There were three differences. Two of them missed. 🙂

Reply:
..., okay I've fixed the typos that I missed.